x/pocketcore/keeper/claim.go

```diff
@@ -49,26 +55,73 @@ func (k Keeper) SendClaimTx(ctx sdk.Ctx, n client.Client, keybase keys.Keybase,
 			return
 		}
 		// send in the evidence header, the total relays completed, and the merkle root (ensures data integrity)
-		if _, err := claimTx(keybase, cliCtx, txBuilder, evidence.SessionHeader, evidence.TotalRelays, root); err != nil {
+		if _, err := claimTx(keybase, cliCtx, txBuilder, evidence.SessionHeader, evidence.NumOfProofs, root, evidenceType); err != nil {
 			ctx.Logger().Error(fmt.Sprintf("an error occured retrieving the coinbase for the claimTX:\n%v", err))
 		}
 	}
 }
 
+func (k Keeper) ValidateClaim(ctx sdk.Ctx, claim pc.MsgClaim) sdk.Error {
+	if claim.EvidenceType == 0 {
+		return pc.NewNoEvidenceTypeErr(pc.ModuleName)
+	}
+	// get the session context
+	sessionContext, er := ctx.PrevCtx(claim.SessionBlockHeight)
+	if er != nil {
+		return sdk.ErrInternal(er.Error())
+	}
+	// if is not a pocket supported blockchain then return not supported error
+	if !k.IsPocketSupportedBlockchain(sessionContext, claim.Chain) {
+		return pc.NewChainNotSupportedErr(pc.ModuleName)
```

Comment:
Why does this error out?

Reply:
Cause you can't create a claim an unsupported blockchain